.specify/bugs/tdd-verify-red-requires-project-flag/issue.md

```
@@ -0,0 +1,40 @@
+- **Issue**: 679
+- **URL**: https://github.com/arrrrny/zuraffa/issues/679
+- **State**: open
+- **Severity**: unknown
+- **Author**: arrrrrny (Ahmet TOK)
+- **Labels**: bug
+
+## Body
+
+When running `zfa tdd verify-red` (and other TDD sub-commands), the user must currently pass `--project` explicitly:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)